backend/modules/auth/router.py

```
@@ -0,0 +1,654 @@
+    )
+
+
+@router.post("/logout")
+async def logout_user(response: Response):
+    """Logout user by clearing HttpOnly cookie"""
+    auth_service.clear_auth_cookie(response)
+    return {"message": "Successfully logged out"}
+
+
+@router.post("/forgot-password")
+async def forgot_password(request: PasswordResetRequest, db: Session = Depends(get_db)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)